contracts/contracts/strategies/NativeStaking/CompoundingValidatorManager.sol

```diff
@@ -36,6 +36,9 @@ abstract contract CompoundingValidatorManager is Governable, Pausable {
     /// to support deposits of 1 ETH.
     uint256 public constant DEPOSIT_AMOUNT_WEI = 32 ether;
 
+    /// @notice Genesis time of the Pectra upgrade
+    uint64 public constant GENESIS_TIME = 1606824023; 
```

Comment:
I'll make this immutable as it'll be different on Hoodi

Reply:
good thinking 👍